ministryofjustice/correspondence_tool_staff
# CT-2987 Remove useless bits and make it migrate data

## Description
Get data and database to migrate upon deployment, **and not break anything meanwhile**.
- seems to work on demo, not breaking anything at least -
## Self-review checklist


### Screenshots
n/a

### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2987

### Deployment
Will break all deployments if flawed. So needs deploying as part of testing.

### Manual testing instructions
Deploy on non-prod environment, and see if app still works. (no migrations, with migrations). Any other suggestions?
